numtrip/numtrip.py

```diff
@@ -0,0 +1,62 @@
+zahlen_matrix = [
+    [2, 4, 16, 8, 8],
+    [4, 26, 8, 2, 1],
+    [4, 4, 16, 4, 2],
+    [2, 8, 1, 4, 1],
+    [2, 4, 4, 4, 4]
+]
+
+#print(zahlen_matrix)
+def obere_zeile():
+    print(f"      1      2      3      4      5    ")
+
+def rand():
+    print(f"  +------+------+------+------+------+")
+
+def leer_zeile():
+    print(f"  ¦      ¦      ¦      ¦      ¦      ¦")
+
+
+def dazwischen_zahl(zahlenliste,nr):
+    print(f"{nr} ¦", end = '')
+    for zahl in zahlenliste:
+        if (zahl<10):
+            print(" ", end = '')
+        print("  ", end = '')
+        if (zahl<0):
+            print(" ",end = '')
+        else:
+            print(zahl, end = '')
+        print("  ¦", end = '')
+    print("")
+
+
+def eingabe():
+    x,y=input("Welches Feld soll geleert werden: ").split()
+    return int(x),int👍
```

Comment:
Das Problem hier: wenn keine Zahlen eingegeben wurden, entsteht hier ein Fehler. Sie sollten erst in einen int umwandeln, wenn `eingabe_überprüfen` ok ist...
```suggestion
    return int(x),int(y)
```